cinema/views.py

```diff
@@ -34,7 +47,32 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
-    def get_serializer_class(self):
+    @staticmethod
+    def params_to_int(string: str) -> list:
+        return [int(str_id) for str_id in string.split(",")]
```

Comment:
You're duplicating this method in 2 viewsets. Move it to a different place and reuse in both cases